ci-operator/jobs/integr8ly/integreatly-operator/integr8ly-integreatly-operator-rhoam-release-v999999.999-presubmits.yaml

```diff
@@ -0,0 +1,394 @@
+presubmits:
+  integr8ly/integreatly-operator:
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - rhoam-release-v999999.999
+    cluster: build01
+    context: ci/prow/e2e
```

Comment:
I have a question mark about this. I do not know what it does and would it be affected by olm type